apple/RNCWebViewManager.m

```diff
@@ -292,4 +282,19 @@ - (BOOL)          webView:(RNCWebView *)webView
   }
 }
 
+RCT_EXPORT_METHOD(releaseWebView:(nonnull NSString *)webViewKey)
+{
+  NSMutableDictionary *sharedWKWebViewDictionary = [[RNCWKWebViewMapManager sharedManager] sharedWKWebViewDictionary];
+  NSMutableDictionary *sharedRNCWebViewDictionary= [[RNCWebViewMapManager sharedManager] sharedRNCWebViewDictionary];
+  
+  sharedWKWebViewDictionary[webViewKey] = nil;
+  
+  RNCWebView *rncWebView = sharedWKWebViewDictionary[webViewKey];
```

Comment:
am i missing something or did you just set this to `nil` on the previous line?

Reply:
Good catch. This should be reading from `sharedRNCWebViewDictionary` instead of `sharedWKWebViewDictionary`

Reply:
Fixed here! https://github.com/discord/react-native-webview/commit/17ba9570fc683b53f3b2c18a9db5805be2d58ffc